docs/docs/about/why-cli.mdx

```diff
@@ -2,8 +2,60 @@
 sidebar_position: 1
 ---
 
-# Why this CLI
+# Why this CLI?
 
 CLI for Microsoft 365 is a cross-platform CLI that allows users on any platform to manage various configuration settings of Microsoft 365.
 
-While building solutions for Microsoft 365 expands beyond the Windows operating system, managing many of the platform settings is possible only through PowerShell on Windows. As more and more users work on non-Windows machines, it's inconvenient for them to have to use a Windows virtual machine to configure their tenants. The CLI for Microsoft 365 allows them to configure their tenants no matter which operating system they use.
+- CLI for Microsoft 365 offers a variety of commands from many different areas of Microsoft 365, such as:
+  - [SharePoint Online](../cmd/spo/spo-get)
+  - [Teams](../cmd/teams/app/app-install)
+  - [Viva engage](../cmd/viva/engage/engage-search)
+  - [Power Apps](../cmd/pa/app/app-export)
+  - [Power Automate](../cmd/flow/flow-disable)
+  - [Power Platform](../cmd/pp/aibuildermodel/aibuildermodel-get)
+  - [OneDrive](../cmd/onedrive/onedrive-list)
+  - [Planner](../cmd/planner/bucket/bucket-add)
+  - [OneNote](../cmd/onenote/notebook/notebook-add)
+  - [Outlook](../cmd/outlook/mail/mail-send)
+  - [SharePoint Embedded](../cmd/spe/containertype/containertype-add)
```

Comment:
Let's order these alphabetically + let's add Entra ID as well since this is quite a large command group.